common/test/acceptance/pages/studio/index.py

```diff
@@ -5,18 +5,31 @@
 from bok_choy.page_object import PageObject
 from . import BASE_URL
 from selenium.webdriver import ActionChains
+from ..common.utils import click_css
 
 
 class DashboardPage(PageObject):
     """
     Studio Home page
     """
 
+    def __init__(self, browser):
+        """
```

Comment:
I don't think `__init__` method is necessary here if we are not performing additional operation in this.